hibernate-search/README.md

```diff
@@ -0,0 +1,12 @@
+# WildFly Swarm - Hibernate Search
```

Comment:
Not sure we need this file. It's not a precendent we've set, and not sure the value it adds as we wouldn't expect users to be trawling through -core source for this information